common.mk

```diff
@@ -28,7 +28,7 @@ install-glide:
 # compesate for the lack of. This conditional tests an environmental variable
 # injected into SEMAPHORE instances, https://semaphoreci.com/docs/available-environment-variables.html
 prep-semaphore:
-	$(SEMAPHORE) && (sudo swapoff -a && sudo dd if=/dev/zero of=/swapfile bs=1M count=8192 && sudo mkswap /swapfile && sudo chmod 0600 /swapfile && sudo swapon /swapfile)
+	if [[ `df -T | grep "/$$" | grep ext` && $(SEMAPHORE) ]]; then sudo swapoff -a && sudo dd if=/dev/zero of=/swapfile bs=1M count=8192 && sudo mkswap /swapfile && sudo chmod 0600 /swapfile && sudo swapon /swapfile; fi
```

Comment:
nit: mind putting the grep expression need to be in single quotes. i'm concerned someone would copy paste into a bash script and it'd break.

Reply:
Curd I missed your comment, will address in another PR